dtfs-central-api/api-tests/v1/utilisation-reports/fee-record-corrections/post-fee-record-correction.api-test.ts

```diff
@@ -0,0 +1,185 @@
+import { HttpStatusCode } from 'axios';
+import {
+  FeeRecordCorrectionTransientFormDataEntityMockBuilder,
+  FeeRecordEntityMockBuilder,
+  PENDING_RECONCILIATION,
+  UtilisationReportEntityMockBuilder,
+} from '@ukef/dtfs2-common';
+import { aTfmUser } from '@ukef/dtfs2-common/mock-data-backend';
+import { withSqlIdPathParameterValidationTests } from '@ukef/dtfs2-common/test-cases-backend';
+import { testApi } from '../../../test-api';
+import { SqlDbHelper } from '../../../sql-db-helper';
+import { mongoDbClient } from '../../../../src/drivers/db-client';
+import { wipe } from '../../../wipeDB';
+import { aPortalUser, aTfmSessionUser } from '../../../../test-helpers';
+
+console.error = jest.fn();
+
+const BASE_URL = '/v1/utilisation-reports/:reportId/fee-records/:feeRecordId/corrections';
+
+describe(`POST ${BASE_URL}`, () => {
+  const getUrl = (reportId: number | string, feeRecordId: number | string) =>
+    BASE_URL.replace(':reportId', reportId.toString()).replace(':feeRecordId', feeRecordId.toString());
+
+  const reportId = 1;
+  const otherReportId = 2;
+  const feeRecordId = 11;
+  const nonExistantFeeRecordId = 12;
+
+  const portalUser = aPortalUser();
+  const portalUserId = portalUser._id.toString();
+
+  const tfmUser = aTfmUser();
+  const tfmUserId = tfmUser._id.toString();
+
+  const uploadedUtilisationReport = UtilisationReportEntityMockBuilder.forStatus(PENDING_RECONCILIATION)
+    .withId(reportId)
+    .withUploadedByUserId(portalUserId)
+    .build();
+
+  const feeRecord = FeeRecordEntityMockBuilder.forReport(uploadedUtilisationReport).withId(feeRecordId).withCorrections([]).build();
+  uploadedUtilisationReport.feeRecords = [feeRecord];
+
+  const otherUtilisationReport = UtilisationReportEntityMockBuilder.forStatus(PENDING_RECONCILIATION)
+    .withId(otherReportId)
+    .withUploadedByUserId(portalUserId)
+    .build();
+
+  const transientFormDataForUserAndFeeRecord = new FeeRecordCorrectionTransientFormDataEntityMockBuilder()
+    .withUserId(tfmUserId)
+    .withFeeRecordId(feeRecordId)
+    .build();
+  const transientFormDataForOtherUser = new FeeRecordCorrectionTransientFormDataEntityMockBuilder().withUserId('abc123').withFeeRecordId(feeRecordId).build();
```

Comment:
How do you feel about moving this (`transientFormDataForOtherUser`) and `transientFormDataForOtherFeeRecord` below into the arrange sections of their respective test cases (as they're only used once)?